src/main/java/commands/AddScheduleCommand.java

```diff
@@ -29,8 +30,8 @@ public void execute(Ui ui, AuthorList authorList) throws TantouException {
         String deadline = parser.getDeadlineDateFromInput(userInput);
 
         if (deadline.isEmpty() || mangaName.isEmpty() || authorName.isEmpty()) {
-            logger.warning("No deadline, author, or manga provided.");
-            throw new TantouException("No deadline date, author, or manga provided!");
+            logger.warning("No schedule date, author, or manga provided.");
+            throw new TantouException("No schedule date, author, or manga provided!");
```

Comment:
I missed this out in the previous PR, but would it be better for the user if we split each case up so the user knows which exactly field they need to provide?